src/components/Modals/Send/hooks/useFormSend/useFormSend.tsx

```diff
@@ -102,10 +103,18 @@ export const useFormSend = () => {
 
         toast({
           title: translate('modals.send.sent', { asset: data.asset.name }),
-          description: translate('modals.send.youHaveSent', {
-            amount: data.cryptoAmount,
-            symbol: data.cryptoSymbol
-          }),
+          description:
+          <Text>
+            <Text>
+              {translate('modals.send.youHaveSent', {
+                amount: data.cryptoAmount,
+                symbol: data.cryptoSymbol
+              })}
+            </Text>
+            <Link href={ data.asset.explorerTxLink } isExternal>
```

Comment:
My mistake, I thought it was draft by default (this is my first PR :)). I still have to test it while sending ETH, waiting on testing funds for gas fees. What do you mean by patches development in this case? And yes, I understand now how the URL must be constructed with the tx ID.

Reply:
Right! Now you have some idea how to construct tx links 😉.
Regarding monkey patches, I meant it you could add some variables effectively forcing/mocking the data coming in to the functions/components and use it in place of real data, which is useful in case e.g you don't want to make an actual tx.
That usually goes hand-in-hand with test coverage (which should always be there anyway except for very specific cases) as a replacement for testing.

Reply:
Regarding drafts. You have the option to make it a draft directly when creating the PR:

![image](https://user-images.githubusercontent.com/17035424/154234165-0d76b580-f8ba-4f82-a96a-5a6b5d05688c.png)

Otherwise, if it's already ready for review but you notice changes needed and don't want this to be on reviewers pile until they're added, you can convert it to draft on that small button under reviewers:

![image](https://user-images.githubusercontent.com/17035424/154234322-9486e230-c4ca-4d8d-9a4a-e3fe12dae7cf.png)